specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,373 @@
+      "properties": {
+        "id": {
+          "description": "The ARM resource id of the long running operation.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the long running operation.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of the long running operation.",
+          "type": "string"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of the resource generated.",
+          "$ref": "#/definitions/DownloadURL"
+        }
+      }
+    },
+    "DownloadURL": {
+      "description": "The URL to download the generated report.",
+      "properties": {
+        "downloadUrl": {
+          "description": "The URL to download the generated report.",
+          "type": "string"
+        },
+        "validTill": {
+          "description": "The time at which report URL becomes invalid/expires in UTC e.g. 2020-12-08T05:55:59.4394737Z.",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "Status": {
+      "description": "The status of the long running operation.",
+      "properties": {
+        "status": {
+          "description": "The status of the long running operation.",
+          "type": "string",
+          "enum": [
+            "InProgress",
+            "Completed",
+            "Failed",
+            "Queued",
+            "NoDataFound",
+            "ReadyToDownload",
+            "TimedOut"
+          ],
+          "x-ms-enum": {
+            "name": "ReportOperationStatusType",
+            "modelAsString": true
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "scopeUsageDetailsParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope associated with usage details operations. This includes '/subscriptions/{subscriptionId}/' for subscription scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, '/providers/Microsoft.Billing/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope. Also, Modern Commerce Account scopes are '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for billingAccount scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billingProfile scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/invoiceSections/{invoiceSectionId}' for invoiceSection scope, and 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/customers/{customerId}' specific for partners.",
+      "x-ms-parameter-location": "method",
+      "x-ms-skip-url-encoding": true
```

Comment:
Does it still make sense to talk about Modern here?

Reply:
Removed the term Modern from all specs